lib/Server.js

```diff
@@ -302,19 +302,28 @@ function Server(compiler, options) {
 	}, this);
 
 	if(options.https) {
-		// for keep supporting CLI parameters
-		if(typeof options.https === "boolean") {
+		// pfx support
+		if(options.pfx) {
```

Comment:
What do you mean? I think users might want to provide a --pfx cli option, - this will end up being on the `options`.